AMM/project/contracts/exchange-contract/src/utils.sw

```diff
@@ -54,22 +49,18 @@ pub fn minimum_output_given_exact_input(
     result_wrapped.unwrap()
 }
 
-pub fn multiply_div(a: u64, b: u64, c: u64) -> u64 {
+pub fn multiply_divide(a: u64, b: u64, c: u64) -> u64 {
```

Comment:
The name of this function doesn't really convey much as to what it is trying to accomplish. I would rename and potentially add some documentation to outline what is the multiplication and division achieving.

Reply:
Cannot come up with a better name now, will rename by the time this is merged to master.